test/e2e/framework/syncer.go

```diff
@@ -127,19 +130,24 @@ func WithDownstreamPreparation(prepare func(config *rest.Config, isFakePCluster
 	}
 }
 
-// Start starts a new syncer against the given upstream kcp workspace. Whether the syncer run
-// in-process or deployed on a pcluster will depend whether --pcluster-kubeconfig and
-// --syncer-image are supplied to the test invocation.
-func (sf *syncerFixture) Start(t *testing.T) *StartedSyncerFixture {
+// Create creates a SyncTarget resource through the `workload sync` CLI command,
+// applies the syncer-related resources in the physical cluster.
+// No resource will be effectively synced after calling this method.
+func (sf *syncerFixture) Create(t *testing.T) *appliedSyncerFixture {
```

Comment:
nit: can we be more descriptive? Is this `CreateSyncTargetAndApplyToDownstream`?

Reply:
That's it yes, and if the name is not too long for you I'm glad to change it.

Reply:
Renamed.